libwayshot/src/lib.rs

```diff
@@ -1,84 +1,51 @@
+//! `libwayshot` is a convenient wrapper over the wlroots screenshot protocol
+//! that provides a simple API to take screenshots with.
+//!
+//! To get started, look at [`WayshotConnection`].
+
 mod convert;
+mod dispatch;
+mod error;
```

Comment:
I think it should be pub mod error

Reply:
Because an error is useful, since we use this error, then the error should be public, then users  can use it in pattern match

Reply:
Oh, I miss it, sorry